extensions/replication/tasks/ReplicateObject.js

```diff
@@ -34,10 +34,14 @@ class ReplicateObject extends BackbeatTask {
     constructor(qp) {
         const qpState = qp.getStateVars();
         super({
-            retryTimeoutS: qpState.repConfig.queueProcessor.retryTimeoutS,
+            retry: qpState.repConfig.queueProcessor.retry,
         });
         Object.assign(this, qpState);
-
+        this.destType = this.destConfig.bootstrapList
+        .find(endpoint => endpoint.site === this.site);
```

Comment:
After finding the site object: e.g.

```
{ "site": "aws-location", "type": "aws_s3" }
```

It looks like we need to also get the `type` from that object.

Reply:
Ah you are right. I should be pulling `type` instead of site.